Support/Testing/CircleCI/Dockerfile

```diff
@@ -40,9 +40,38 @@ MAINTAINER Stephane Sezer <sas@fb.com>
 RUN apt-get update
 RUN apt-get install -y software-properties-common python-software-properties wget
 
+# Incase lldb needs built
```

Comment:
The "utilities" should be in a separate PR and should be clearly marked as optional stuff. Did you check to see if it was possible to just have a command-line flag and enable these when developers need it only?

Also, Incase -> In case.